src/infill.cpp

```diff
@@ -201,14 +201,19 @@ void Infill::multiplyInfill(Polygons& result_polygons, Polygons& result_lines)
         }
     }
     result.add(first_offset);
-    Polygons reference_polygons = first_offset;
-    for (size_t infill_line = 1; infill_line < infill_multiplier / 2; infill_line++) // 2 because we are making lines on both sides at the same time
+    if (infill_multiplier > 3)
```

Comment:
I'd like to see some documentation why infill_multiplyer > 3 is important here.

Reply:
will add it